### PR TITLE
[SO-6] Add SOLN_META message for Fuzed Wagon

### PR DESCRIPTION
**Pull request by GuillaumeDec**
https://swift-nav.atlassian.net/browse/SO-6

### CODE REVIEW

File: spec/yaml/swiftnav/sbp/solution_meta.yaml [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+                 desc: The type of sensor
+                 values:
+                   - 0: GNSS Position (see GNSSInputType)
+                   - 1: GNSS Velocity Displacement (see GNSSInputType)
+                   - 2: GNSS Velocity Doppler (see GNSSInputType)
+                   - 3: Odometry Ticks (see OdoInputType)
+                   - 4: Odometry Speed (see OdoInputType)
+                   - 5: IMU Sensor (see IMUInputType)
+                   - 6: Reserved
+                   - 7: Reserved
```
Comment (scarcanague) [inline spec/yaml/swiftnav/sbp/solution_meta.yaml:146]: 2 more sensors seems a little tight here, Im wondering if we should rather to 0-3 for type of sensor, 4-5 for description and 6-7 for reserved

File: spec/yaml/swiftnav/sbp/solution_meta.yaml [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+     short_desc: Solution Sensors Metadata
+     public: true
+     desc: |
+       This message contains all metadata about the sensors received and/or used in computing the Fuzed Solution.
+       It focuses primarly, but not only, on GNSS metadata.
+     fields:
+       - pdop:
+           type: u16
+           units: 0.01
+           desc: Position Dilution of Precision, as per last received DOPS,
```
Comment (scarcanague) [inline spec/yaml/swiftnav/sbp/solution_meta.yaml:171]: I would change the phrasing a bit and keep it short:
`Position Dilution of Precision, as per last available DOPS from Starling GNSS engine`

File: spec/yaml/swiftnav/sbp/solution_meta.yaml (hunk 1 above)
Comment (GuillaumeDec) [reply spec/yaml/swiftnav/sbp/solution_meta.yaml:146]: Good point, Done!

File: java/src/com/swiftnav/sbp/client/MessageTable.java [hunk 3]
```diff
@@ -173,6 +173,10 @@
 import com.swiftnav.sbp.settings.MsgSettingsReadByIndexDone;
 import com.swiftnav.sbp.settings.MsgSettingsRegister;
 import com.swiftnav.sbp.settings.MsgSettingsRegisterResp;
+import com.swiftnav.sbp.solution_meta.Gnssinputtype;
```
Comment (silverjam) [inline java/src/com/swiftnav/sbp/client/MessageTable.java:176]: We need to fix the java generator so that it correctly camel cases these embedded types.

Comment (GuillaumeDec) [reply java/src/com/swiftnav/sbp/client/MessageTable.java:176]: Good catch @silverjam ! Thanks!

Comment (GuillaumeDec) [reply java/src/com/swiftnav/sbp/client/MessageTable.java]: Done! 👍

**Pull request merged**